AT_01.15.05 | Start Page > Register Agent Negative > Error message is displayed when trying to register with invalid email

EkaterinaRomankova opened the pull request:
https://trello.com/c/Mgn4JhFH/293-at011505-start-page-register-agent-negative-error-message-is-displayed-when-trying-to-register-with-invalid-email

elviraKh1 reviewed (commented):
File cypress/e2e/testUiAndFunction/startPage/.DS_Store doesn't belong to structure of the project

EkaterinaRomankova commented:
I don't use this file

elviraKh1 commented:
> I don't use this file
Yes,  I see, and it shouldn't be in your  PR.. For your AT you need only 2 files, please remove .DS_Store from PR.

EkaterinaRomankova commented:
> > I don't use this file
> > Yes,  I see, and it shouldn't be in your  PR.. For your AT you need only 2 files, please remove .DS_Store from PR.

I can't delete this file, because i added fixture on the first commit after i saw that we had already what i need. On second commit i deleted my fixture in that file

elviraKh1 commented:
- It's your choice you can close PR and start a new branch, can use https://chrome.google.com/webstore/detail/refined-github/hlepfoohegkhhmjieoechaddaejaokhf/related?hl=en-US 
I can suggest reading about the command git rm  :
git rm cypress/e2e/testUiAndFunction/startPage/.DS_Store --cached
git status
git commit -m ".DS_Store was removed"
git push

EkaterinaRomankova commented:
> you can (in my opinion - I can be wrong! )
> 
> * close PR and start a new branch,
> * can use https://chrome.google.com/webstore/detail/refined-github/hlepfoohegkhhmjieoechaddaejaokhf/related?hl=en-US
> 
> ![image](https://user-images.githubusercontent.com/115122344/214431828-369622dc-7dfb-4169-b266-22490b30377d.png)
> 
> * I can suggest reading about the command git rm  :
>   git rm cypress/e2e/testUiAndFunction/startPage/.DS_Store --cached
>   git status
>   git commit -m ".DS_Store was removed"
>   git push
> * or something else

I won't delete the commit, it's not a bug. and I don't see items for pics on my file
